mailet/standard/src/main/java/org/apache/james/transport/mailets/LogMessage.java

```diff
@@ -101,26 +97,25 @@ public String getMailetInfo() {
 
     @Override
     public void service(Mail mail) {
-        logComment();
+        String logs = "";
```

Comment:
Or we can use a stringBuilder here that we build at the end of the request to avoid variable re-allocation?